code/index.html

```diff
@@ -1,30 +1,91 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Luksanmäki-MTB</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+  <header>
+    <picture>
+      <source srcset="./mtb3.jpg" media="(max-width: 600px)" />
+      <source srcset="./mtb4.jpg" media="(max-width: 900px)" />
+      <img class="hero-image" src="./mtb5.jpg" alt="mtb image" />
+    </picture>
+    <div class="title">
+      <h1>Luksanmäki-MTB</h1>
+    </div>
+  </header>
+
+  <div class="about">
+    <p>Luksanmäki-MTB is a group of eager mountain bikers who love challenges and mud.</p>
+  </div>
+
+  <!-- sign up form -->
+  <div class="grid-container">
+    <div class="grid-item">
+      <h2>Join Us!</h2>
+      <form class="form" action="https://httpbin.org/anything" method="POST">
+        <div class="user-information">
+          <label for="firstName">First name:</label>
+          <input name="firstName" type="text" id="firstName" placeholder="Your first name" /><br>
+          <label for="lastName">Last name:</label>
+          <input name="lastName" type="text" id="lastName" placeholder="Your last name" /><br>
+          <label for="email">Email:</label>
+          <input name="email" type="email" id="email" required placeholder="your@email.com" />
+        </div>
+        <div class="radiobuttons">
+          <h3>Which one do you prefer?</h3>
+          <input type="radio" name="bike" value="trek" />
+          <label for="trek">TREK</label>
+          <input type="radio" name="bike" value="cannondale" />
+          <label for="cannondale">Cannondale</label>
+          <input type="radio" name="bike" value="other" />
+          <label for="other">Other</label>
+        </div><br>
```

Comment:
We shouldn't use `<br />` tags. As a matter of fact, you don't need them here, because you use `<div />` tag, which already has feature to break line. Let's delete them